CONTRIBUTORS.txt

```diff
@@ -101,7 +101,7 @@ Systopia - Björn Endres, Thomas Leichtfuss
 Tadpole Collective - Kevin Cristiano
 Tanya Bouman
 Team Expansion - Greg Harris
-Third Sector Design - Michael McAndrew
+Third Sector Design - Michael McAndrew, Kurund, Jalmi, William Mortada
```

Comment:
There's a typo: can you remove the comma between Kurund and Jalmi?

Reply:
Sorry! Fixed it now.